README.md

```diff
@@ -30,6 +30,21 @@ requests need to be manually reviewed and merged by a human.
 It is ***highly recommended*** to also use [TagBot][tagbot], which automatically tags a release in your
 repository after the new release of your package is merged into the registry.
 
+### Requirements for new packages
+
+New packages that meet the following requirements will be eligible for automatic merging (see below).
+Packages that do not meet these requirements will be subject to a longer period of community review.
+These requirements have been established to ensure that packages that are available for download via
+the General registry are well-documented and sufficiently tested for reliable use.
+
+- A `README.md` file of at least 100 words that explains the purpose of the package and contains example usage;
+- A set of user documentation that provides, at a minimum, details on the package's exported methods and
+structures; and
+- A test suite that covers the main features of the package. Code coverage of at least 50% is required,
```

Comment:
I don't know that it's practical to require a specific amount of coverage; some packages may not be able to run on CI since they require particular software to be installed (Matlab.jl or similar comes to mind, as do the GPU packages).

Reply:
The challenge is to have an objective set of criteria. Just saying "have some tests" isn't going to be effective, I don't think.

This doesn't mean that these packages won't be merged; it means that there has to be some discussion around why they can't meet these requirements. I think that's a valuable discussion to have regardless.

Reply:
I think code coverage is an awful metric and I never use Codecov anymore, I personally do think "have some tests" is the most we can ask here.

Reply:
I personally use CodeCov a lot and like code coverage as a metric (it's not great, but it's something). But I don't think we can force this on others.

I would also say let's take out the code coverage part.